### DISCUSSION TITLE
How to copy the log

### DISCUSSION THREAD

**Discussion by Mataingarix** [victornpb/undiscord#611] [1. Questions & Answers]
I am using this script on mobile with Kiwi, mobile is very laggy, very limited in terms of the amount of text that can be copied. I had deleted 1k messages and tried a bunch of times to select the text properly, until I succeeded, only for my phone to return "failed to copy text" I looked around in the UI, there's absolutely no button anywhere, I even tried pressing the ≡ it lagged for a few seconds so i pressed it again, only to end up having the clear log button be pressed thanks to the awesome input lag from Android lol
Is there even a location where it gets stored? Can it be recovered? I tried looking around and literally no one ever asks this

**Comment by viiper2000** [comment]
Undiscord doesn't store that data, or any personal data from Discord for that matter, or at least that's what victornpb says in the security policy. 

Once it's gone off your screen, it's probably gone for good.
